react-sdk/src/providers/tambo-client-provider.tsx

```diff
@@ -36,6 +47,8 @@ export interface TamboClientContextProps {
   queryClient: QueryClient;
   /** Whether the session token is currently being updated */
   isUpdatingToken: boolean;
+  /** Additional headers to include in all requests */
```

Comment:
I think the name `defaultHeaders` implies that it is all the existing header or something, rather than extra headers that we are adding to the existing (default) headers. 

How about `additionalHeaders` or `extraHeaders`